From 3744aa1990f9abf7aa9887261193587d5c48ce2a Mon Sep 17 00:00:00 2001
From: hubshuffle <hubshuffled@gmail.com>
Date: Fri, 5 Apr 2019 22:40:22 +0100
Subject: [PATCH] wip

---
 app.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/app.py b/app.py
index d3c9203..2acf3ac 100644
--- a/app.py
+++ b/app.py
@@ -7,8 +7,8 @@ app = Flask(__name__)
 
 @app.route('/')
 def hello_world():
-    return 'Hello World!'
+    return 'Hello World! Package'
 
 
 if __name__ == '__main__':
-    app.run(host='0.0.0.0')
+    app.run(host='0.0.0.0', port=7777)
-- 
2.21.0
